jscomp/main/bsb_main.ml

```diff
@@ -195,7 +200,11 @@ let () =
           -> (* -make-world all dependencies fall into this category *)
           begin
             Arg.parse_argv bsb_args bsb_main_flags handle_anonymous_arg usage ;
-            let config_opt = Bsb_ninja_regen.regenerate_ninja ~generate_watch_metadata:true ~override_package_specs:None ~not_dev:false cwd bsc_dir ~forced:!force_regenerate in
+            let config_opt = Bsb_ninja_regen.regenerate_ninja 
+                ~generate_watch_metadata:true 
+                ~override_package_specs:None 
+                ~not_dev:false cwd bsc_dir 
```

Comment:
@bobzhang shouldn't `not_dev`be `true` here to trigger the quiet mode?

Reply:
The name is a bit misleading, it should be called `not_top`..

Reply:
hah! gotcha, that makes sense